python/paddle/fluid/layers/nn.py

```diff
@@ -5494,11 +5521,13 @@ def sequence_pad(x, pad_value, maxlen=None, name=None):
 
 def sequence_unpad(x, length, name=None):
     """
-    **Sequence Unpad Layer**
+    **Note**:
+    
+    **The input of the OP is Tensor and the output is LoDTensor.**  
+     
+    The OP removes the filled element from the input based on the length information and returns a LoDTensor.
 
-    This layer removes the padding data in the input sequences and convert
-    them into sequences with actual length as output, identitied by lod
-    information.
+    On the contrary, the OP of sequence_pad can fill the data. See: :ref:`api_fluid_layers_sequence_pad`   
```

Comment:
On the contrary, the OP of sequence_pad can fill the data. 
=>
For padding operation,

Reply:
done